### PR TITLE
fix: namespace CSS to `bio-improved-canvas`

### PR DESCRIPTION
**Pull request by philippfromme**
* changes styling to depend on module loaded instead of CSS existing

---

Related to https://github.com/camunda/web-modeler/pull/7847

### CODE REVIEW

File: assets/high-contrast-canvas.css [hunk 1]
```diff
@@ -8,33 +8,18 @@
...
-/* Properties Panel */
-
-.bio-properties-panel-header {
-  background-color: white;
-}
-
-/* Keep forms white */
-.cfp-container .bio-properties-panel-header {
-  background-color: var(--header-background-color);
-}
```
Comment (marstamm) [inline assets/high-contrast-canvas.css:30]: Great to get rid of this hack

File: assets/high-contrast-canvas.css [hunk 2]
```diff
@@ -8,33 +8,18 @@
...
+.dmn-drd-container.bio-improved-canvas {
   --dmn-definitions-background-color: white;
   --dmn-definitions-border-color: var(--gray-20);
 }
 
-/* Properties Panel */
-
-.bio-properties-panel-header {
-  background-color: white;
-}
```
Comment (marstamm) [inline assets/high-contrast-canvas.css:25]: Can we safely get rid of this? Before the change, the top was white:

image

Now it is Grey
image

Comment (marstamm) [reply assets/high-contrast-canvas.css:25]: We might need to add namespaces to the properties panel container as well

File: assets/high-contrast-canvas.css (hunk 1 above)
Comment (philippfromme) [reply assets/high-contrast-canvas.css:30]: I was surprised to find this. 😅 All web modeler style changes have been moved to web modeler.

File: assets/high-contrast-canvas.css (hunk 2 above)
Comment (philippfromme) [reply assets/high-contrast-canvas.css:25]: This is now adjusted by the web modeler itself.

Comment (philippfromme) [reply assets/high-contrast-canvas.css:25]: Also because we can't namespace it from inside the canvas and it needs to be applied conditionally.

Comment (marstamm) [reply assets/high-contrast-canvas.css:25]: Good to go from my side then 👍

**Review by marstamm — APPROVED** [review]

**Pull request merged**